sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getCompletedApplicationDetails.e2e-spec.ts

```diff
@@ -908,6 +914,217 @@ describe("ApplicationStudentsController(e2e)-getCompletedApplicationDetails", ()
     },
   );
 
+  it("Should get application with change request details when there is an in-progress change request pending Ministry approval.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(
+      appDataSource,
+      { student: sharedStudent, msfaaNumber: sharedSignedMSFAANumber },
+      { applicationStatus: ApplicationStatus.Completed },
+    );
+    const [firstDisbursement] =
+      application.currentAssessment.disbursementSchedules;
+    await disbursementScheduleRepo.save(firstDisbursement);
```

Comment:
Looks like this save operation does not have any effect.

Reply:
Removed, nice catch.